pkg/chains/signing.go

```diff
@@ -36,11 +37,12 @@ import (
 	"knative.dev/pkg/logging"
 )
 
+// TODO: Unclear why an interface is defined here.
```

Comment:
Looks like these are used in the reconcilers.

Reply:
Removed this comment